Frontend/ui-customer/src/Screens/Types.purs

```diff
@@ -570,14 +570,21 @@ type Contact = {
      phoneNo :: String
 }
 
+data RateCardType = ViewStart | ViewDriverAddition | ViewFareUpdate
+derive instance genericRateCardType :: Generic RateCardType _
+instance showRateCardType :: Show RateCardType where show = genericShow
+instance eqRateCardType :: Eq RateCardType where eq = genericEq
+
 type RateCard =
   {
     baseFare :: Int,
     extraFare :: Int,
     pickUpCharges :: Int,
     additionalFare :: Int,
     nightShiftMultiplier :: Number,
-    nightCharges :: Boolean
+    nightCharges :: Boolean,
+    toggleRateCardView :: RateCardType,
```

Comment:
what does toggleRateCardView does?

Reply:
I changed the name to currentRateCardType after discussing it with Vignesh bhaiya